application_service/app_service.cc

```diff
@@ -181,7 +181,7 @@ bool measure_binary(const string &file, string *m) {
   }
   byte         digest[32];
   unsigned int len = 32;
-  if (!digest_message("sha256", file_contents, bytes_read, digest, len)) {
+  if (!digest_message(digest_sha256, file_contents, bytes_read, digest, len)) {
```

Comment:
As an example, this diff would appear like: `if (!digest_message(Digest_sha256, ...` 

with the suggested global variable naming convention.

Reply:
You can stop reviewing the diffs here onwards, as these are all generated by tooling.